keyboards/centromere/keymaps/mini/keymap.c

```diff
@@ -0,0 +1,161 @@
+#include QMK_KEYBOARD_H
+
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
+#define _ADJUST 16
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  LOWER,
+  RAISE,
+  ADJUST,
+};
```

Comment:
```suggestion
// Defines for layer movement
#define LOWER MO(_LOWER)
#define RAISE MO(_RAISE)
```

`QWERTY` and `ADJUST` keycodes aren't actually used in the keymap, and I made another suggestion below that should simplify the ADJUST functionality.